coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,9 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter, available_filters):
```

Comment:
I think we should pass the list of all the filters instead of passing it in string format. Makes more sense to format it over here.